application/src/main/java/org/thingsboard/server/service/install/SqlDatabaseUpgradeService.java

```diff
@@ -449,38 +453,35 @@ public void upgradeDatabase(String fromVersion) throws Exception {
             case "3.2.2":
                 try (Connection conn = DriverManager.getConnection(dbUrl, dbUserName, dbPassword)) {
                     log.info("Updating schema ...");
+                    conn.createStatement().execute("CREATE TABLE IF NOT EXISTS resource ( " +
```

Comment:
Why this lines are affected by your PR?

Reply:
@ashvayka  I had deleted redundant "try-catch"

Reply:
They are not redundant. Don't touch the code that is not related to your changes, please. This block merge of main changes.